.env.betanet

```diff
@@ -0,0 +1,12 @@
+# BetaNet env file
+
+# TODO: Below variabls are of privatenet. Update it with betanet env variables
```

Comment:
why there is TODO here?

Reply:
We have not deployed the app yet. The env variable values for betanet is going to be different.